samples/file-encryptor/enclave/openssl_src/encryptor.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <string.h>
+
+#include "common/trace.h"
+#include "encryptor.h"
+
+ecall_dispatcher::ecall_dispatcher() : m_encrypt(true), m_header(NULL)
+{
+}
+
+int ecall_dispatcher::initialize(
+    bool encrypt,
+    const char* password,
+    size_t password_len,
+    encryption_header_t* header)
+{
+    int ret = -1;
+    TRACE_ENCLAVE(
+        "ecall_dispatcher::initialize : %s request",
+        encrypt ? "encrypting" : "decrypting");
+
+    if (header == NULL)
+    {
+        TRACE_ENCLAVE("initialize() failed as NULL was passed in place of "
+                      "(encryption_header_t *)");
+        goto exit;
+    }
+
+    m_password = std::string(password, password + password_len);
```

Comment:
encryptor.h has using namespace std already. Can you also update this in the Mbed TLS implementation?
```suggestion
    m_password = string(password, password + password_len);
```

Reply:
implemented suggestion.